draft-ietf-sedate-datetime-extended.md

```diff
@@ -592,6 +591,8 @@ Note that a `time-zone` is syntactically similar to a `suffix-tag`, but
 does not include an equals sign.
 This special case is only available for time zone tags.
 
+Also note that `time-zone-name` MUST be the name of an IANA Time Zone.
```

Comment:
This is a difficult statement, as it appears to require checking something that may be in an inconsistent state inconsistent between generator and recipient.